protocols/v2/subprotocols/template-distribution/src/lib.rs

```diff
@@ -1,23 +1,18 @@
+//! # Stratum V2 Template Distribution Protocol Messages Crate
+//!
+//! The following crate contains the messages defined in the Template Distribution Protocol of
+//! Stratum V2.  The Template Distribution protocol can be used to receive updates of the block
+//! template to use in mining.
+//!
+//! ## Build Options
+//! This crate can be built with the following features:
+//! - `std`: Enables support for standard library features.
+//! - `with_serde`: Enables support for serialization and deserialization using Serde.
```

Comment:
```suggestion
//! - `with_serde`: Enables support for serialization and deserialization using Serde. Note that this feature flag is only used for the Message Generator, and deprecated for any other kind of usage. It will likely be fully deprecated in the future.
```

Reply:
Have you missed this one? It's required, otherwise it won't be rendered on rust docs